### PR TITLE
feat: add project to contest

### LINKED ISSUE #100: "Participating in contest" badge
screen shot 2019-02-01 at 10 27 22

### LINKED ISSUE #140: Remove email field from contest term modal
The modal should only serve the purpose of accepting the terms and conditions for the contest

### PR DESCRIPTION
**Pull request by nicosantangelo**
Closes #100 #140 #141

### CODE REVIEW

File: src/components/HomePage/HomePage.types.ts [hunk 1]
```diff
@@ -4,8 +4,11 @@ import { NavigateToAction } from 'decentraland-dapps/dist/modules/location/actio
 import { ProjectState } from 'modules/project/reducer'
 import { createProjectFromTemplate, CreateProjectFromTemplateAction } from 'modules/project/actions'
 
-export type Props = {
+export type DefaultProps = {
   projects: ProjectState['data']
+}
+
+export type Props = DefaultProps & {
```
Comment (cazala) [inline src/components/HomePage/HomePage.types.ts]: i think this should be `Partial<DefaultProps>`, since they are not actually required

**Comment by cazala** [comment]
🥇 

Just one thing: if I go to "Update Your Entry" or "Add To Contest" and I change the values of the form, and then I hit cancel, the changes are still saved, as if I clicked on submit.

Nice to have: if the users is signed in, it would be nice to pre-populate the "Eth Address" field with his current account address

File: src/components/HomePage/HomePage.types.ts (hunk 1 above)
Comment (nicosantangelo) [reply src/components/HomePage/HomePage.types.ts]: [Explained here!](https://github.com/decentraland/builder/pull/160#issuecomment-462733310)

**Comment by nicosantangelo** [comment]
> Just one thing: if I go to "Update Your Entry" or "Add To Contest" and I change the values of the form, and then I hit cancel, the changes are still saved, as if I clicked on submit.

Whops, on it. I'll fix conflicts too

> Nice to have: if the users is signed in, it would be nice to pre-populate the "Eth Address" field with his current account address

I thought of this too, but because we won't add "wallet" support until after the contest is finished I don't think we'll be able to do it

**Comment by nicosantangelo** [comment]
Updated!

**Pull request merged**